packages/calypso-e2e/src/lib/flows/start-site-flow.ts

```diff
@@ -43,20 +45,38 @@ export class StartSiteFlow {
 		await this.page.click( selectors.button( text ) );
 	}
 
+	/**
+	 * Enter site vertical.
+	 *
+	 * @param {string} vertical Name of the vertical to select
+	 */
+	async enterVertical( vertical: string ): Promise< void > {
+		await this.page.waitForLoadState( 'networkidle' );
+
+		const input = this.page.locator( selectors.verticalInput );
+		await input.fill( vertical );
+
+		const readBack = await input.inputValue();
+		if ( readBack !== vertical ) {
+			throw new Error( `Failed to set vertical: expected ${ vertical }, got ${ readBack }` );
+		}
+	}
```

Comment:
**praise**: this is good implementation, very similar to what I would do 👍

Reply:
I think I just copied the pattern from a method you probably wrote in the same file 😂